mui/mui.py

```diff
@@ -40,14 +49,40 @@ def __init__(self, find: Set[int], avoid: Set[int], view: BinaryView):
     def run(self):
         """Initializes manticore, adds the necessary hooks, and runs it"""
 
-        # clear state UI
+        bv = self.view
+
+        # set up state and clear UI
         state_widget: StateListWidget = widget.get_dockwidget(self.view, StateListWidget.NAME)
-        state_widget.notifyStatesChanged({})
+
+        if bv.session_data.mui_state is None:
+            bv.session_data.mui_state = MUIState(bv)
+            state_widget.listen_to(bv.session_data.mui_state)
+
+        bv.session_data.mui_state.notify_states_changed({})
+
+        settings = Settings()
+        prefix = "mui.run."
```

Comment:
Hmmm. Thinking out loud: So maybe instead of a class constant, this should be a module constant. I think those would fit best in a new file called `constants.py`?